omnibus/files/private-chef-cookbooks/private-chef/recipes/post_11_upgrade_cleanup.rb

```diff
@@ -65,16 +65,3 @@
 end
 
 private_chef_package_cleaner "varnish"
-
-# Remove old PostgreSQL data directory
-#
-# pg_upgrade will helpfully generate a cleanup script which removes
-# the previous database cluster.  This is nice, because we don't have
-# to worry about keeping track of the location of the old cluster.
```

Comment:
Removing this  gets messy because of enterprise/private/old-oss chef server installations that could be getting upgraded from.
I'm concerned that the cleanup of the old data will no longer happen - because the postgres data was not always in a versioned directory IIRC - there may be older install that have the path as /var/opt/opscode/postgresql/data instead of postgresql/9.2/data.

Perhaps updating the new `postgres_upgrade_cleanup` to handle both would work?

This might also merit a modification of the upgrade pipeline to verify that old directories are removed.